docs/pages/versions.js

```diff
@@ -76,5 +76,5 @@ Page.getInitialProps = async () => {
     });
   }
 
-  return { versions: sortedUniqBy(versions, 'version') };
+  return { versions: uniqBy(sortBy(versions, 'version'), 'version') };
```

Comment:
I don't think `sortBy` is necessary, `uniqBy` seems to take a function as second argument

```suggestion
  return { versions: uniqBy(versions, item => item.version) };
```